### DISCUSSION TITLE
Mqtt from different network

### DISCUSSION THREAD

**Discussion by calasci** [syssi/esphome-jk-bms#465] [General]
I have a camper network BMS, and home assistant on the home network, how do I use mqtt? Thank you

**Comment by syssi** [comment]
You could make your MQTT server available via the internet. So it can be accessed from anywhere. You could also use the `wireguard` ESPHome component to setup a VPN connection to your second network.

**Comment by calasci** [comment]
Io ho già moduli con mqtt funzionanti , solo jkbms- esphome ha problemi grazie
